create-snowpack-app/cli/createSnowpackApp.js

```diff
@@ -23,7 +23,8 @@ function hasPmInstalled(packageManager) {
 }
 
 function validateArgs(args) {
-  const {template, useYarn, usePnpm, force, target, _} = yargs(args);
+  const {template, useYarn, usePnpm, force, target, install, _} = yargs(args);
+  const toInstall = install == null || install && install !== 'false';
```

Comment:
yargs should parse `--install` & `--no-install` into a boolean value, so you should be able to do:

```
 const toInstall = install !== undefined ? install : true;
```

Reply:
Unfortunately this isn't true -- I've found most users try `--install false` (or similar). This logic handles all those correctly

Reply:
Ha, I hadn't thought of that. Fair point.

I appreciate the smarter one-liner, but let's stick to the more straightforward `const toInstall = install !== undefined ? install : true;` for now. Since we don't do `--foo false` handling on any of our other flags, I'd rather keep our handling consistent across all flags for now.